roles/docker/templates/docker.service.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Docker Application Container Engine
+Documentation=http://docs.docker.com
+After=docker.socket early-docker.target network.target
+Requires=docker.socket early-docker.target
+
+[Service]
+EnvironmentFile=-/run/flannel_docker_opts.env
```

Comment:
This is just a copy of /usr/lib/systemd/system/docker.service as suggested in here https://coreos.com/os/docs/877.1.0/customizing-docker.html we can inject drop it settings just by adding files in ``` /etc/systemd/system/docker.service.d/ ```